### PR TITLE
Id validation

### PR DESCRIPTION
**Pull request by nevensh**
hey Elias

I've finished building the validation of  id number algorithm  .
please check my new pull request .
- I applied all the required points 
-  I've already done a great job. 
- And it's really needs a little thought.

thank you ^_^

### CODE REVIEW

File: Email-validation/script.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+
+function EmailValidation (Email)
+{
+    const template= /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (EliasSalom) [inline Email-validation/script.js]: please explain to me if you have been knowing what this line exactly is?

**Review by EliasSalom — COMMENTED** [review]
please check my comment and answer it!

File: Email-validation/script.js (hunk 1 above)
Comment (nevensh) [reply Email-validation/script.js]: hey Elias 
I was answered you in another issue related in Email validation.
 But I don't understand how this is , I pressed here to do pull request related in Id validation .
Did you check my new branch (Id validation ) ?!

Comment (EliasSalom) [reply Email-validation/script.js]: image

**Review by EliasSalom — APPROVED** [review]
Good work

**Pull request merged**